### PR TITLE
Feature EPL-1476: Icon buttons.

### PR DESCRIPTION
**Pull request by leandroallemandismf**

### CODE REVIEW

File: packages/ComponentLibrary/src/components/IconButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  ...props
+}) => {
+  return (
+    <MUIIconButton {...props}>
+      <img src={icon} alt={alt} />
+    </MUIIconButton>
+  );
+};
+
+export default IconButton;
```
Comment (sjcuello) [inline packages/ComponentLibrary/src/components/IconButton/index.tsx:17]: Missing space

File: packages/ComponentLibrary/src/components/IconButton/types.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { IconButtonProps } from '@mui/material';
+
+export interface IIconButton extends IconButtonProps {
+  /**
+   * The URL or path of the icon to be displayed.
```
Comment (sjcuello) [inline packages/ComponentLibrary/src/components/IconButton/types.ts:5]: Delete comments unnessesary

File: packages/ComponentLibrary/src/components/IconButton/types.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+   * @example "/path/to/icon.png"
+   */
+  icon: string;
+
+  /**
+   * The alternative text for the icon. This is optional.
+   * @default "icon"
+   */
+  alt?: string;
+}
```
Comment (sjcuello) [inline packages/ComponentLibrary/src/components/IconButton/types.ts:15]: Missing space

File: packages/MainUI/src/screens/Home/index.tsx [hunk 4]
```diff
@@ -116,4 +129,4 @@ const Home = () => {
   );
 };
 
-export default Home;
+export default Home;
```
Comment (sjcuello) [inline packages/MainUI/src/screens/Home/index.tsx:132]: Missing space

**Review by sjcuello — APPROVED** [review]

**Review by Sankku — APPROVED** [review]

**Pull request merged**